scripts/pre-commit.sh

```diff
@@ -82,3 +82,34 @@ if [ $SHOULD_FAIL_PRECOMMIT -ne 0 ]; then
 else
   verify_no_included_apikeys
 fi
+
+
+# Function to concatenate CircleCI configuration files
```

Comment:
I'm generally allergic to bash, I might study porting all the logic into ruby at some point